src/components/OSCALDrawerSelector.js

```diff
@@ -95,6 +96,13 @@ function createTree(backendUrl, handleClose) {
 
 export default function OSCALDrawerSelector(props) {
   const theme = useTheme();
+  const oscalObjectUuid = useParams()?.id ?? "";
+
+  useEffect(() => {
+    if (!oscalObjectUuid) {
+      props.handleOpen();
+    }
+  }, [oscalObjectUuid]);
```

Comment:
This has the weird consequence of opening the `drawer` upon any load. I think it is because `uuid` is not guaranteed to be set in time of the drawer being created?

Reply:
Upon looking at this more, it seems that `useParams().id` is not defined. Which is odd because I can get the `Loader` to correctly print the `uuid`.

Reply:
There is some issue with the complexity of `OSCALLoader` and it's communication to `App.js`, where `App.js` is unable to grab `useParams()`. You're right about the `uuid` not being set in time. Callback functions like `useEffect()` can be used to capture states after render, although `useParams()` is a hook itself and cannot be used within another hook.